pom.xml

```diff
@@ -554,32 +593,91 @@
         <artifactId>maven-enforcer-plugin</artifactId>
         <executions>
           <execution>
-            <id>enforce-maven</id>
+            <id>display-info</id>
+            <phase>validate</phase>
             <goals>
+              <goal>display-info</goal>
               <goal>enforce</goal>
             </goals>
             <configuration>
               <rules>
                 <requireMavenVersion>
-                  <version>(,2.1.0),(2.1.0,2.2.0),(2.2.0,)</version>
-                  <message>Build with Maven 2.2.1 or Maven 3.0.4 (or later). Maven 2.1.0 and 2.2.0 produce incorrect GPG signatures and checksums respectively.</message>
-                </requireMavenVersion>
-                <requireMavenVersion>
-                  <version>(,3.0),[3.0.4,)</version>
-                  <message>Build with Maven 3.0.4 or later. Maven 3.0 through 3.0.3 inclusive do not pass correct settings.xml to Maven Release Plugin.</message>
+                  <version>[3.3.9,)</version>
+                  <message>3.3.9 is required at least.</message>
```

Comment:
BTW was there any specific reason for this bump, or just to be on the safe side?

Reply:
See the discission in https://github.com/jenkinsci/pom/pull/14#discussion_r133283276 . No specific reason for this version